packages/css/baseline/ds-reset.css

```diff
@@ -2,13 +2,16 @@
 
 /* https://wiki.csswg.org/ideas/mistakes */
 
+/* TODO: Can this be moved to the related components? */
 [class^='ds-'],
 [class^='ds-']::before,
 [class^='ds-']::after {
   box-sizing: border-box;
 }
 
 /* Inherit fonts for inputs and buttons */
+
+/* TODO: Can this be moved to the related form element CSS files? */
```

Comment:
Yeah I think this is a better candididate for moving to to their relevant components.
However, it gives us another point of failure

Reply:
See comment above, and I suggest forgetting to add styling should be done in reviews, not by an "invisible" dependency relation :)